src/package.cmd.js

```diff
@@ -12,14 +12,15 @@
 const chalk = require('chalk');
 const glob = require('glob');
 const path = require('path');
-const fs = require('fs-extra');
+const fs = require('fs');
 const ProgressBar = require('progress');
-const archiver = require('archiver');
```

Comment:
@koraa As discussed in slack, please put the `archiver` -> `yazl` changes in a separate issue and pull request. this is an unrelated change to this topic, and if it is really faster, we could already benefit from the replacement.